tests/test_rand.py

```diff
@@ -13,25 +13,31 @@
 
 from OpenSSL import rand
 
-from .util import NON_ASCII, TestCase
+from .util import NON_ASCII, tmpfile
```

Comment:
This reminds me that fixtures should generally go into `contest.py`.  Is it possible, that pytest has different discovery mechanisms?